docs/source/gmaps.rst

```diff
@@ -221,7 +221,9 @@ Markers are currently limited to the Google maps style drop icon. If you need to
 
     gmaps.configure(api_key="AI...")
 
-    starbucks_locations = gmaps.datasets.load_dataset("starbucks_uk")
+    starbucks_kfc_locations = gmaps.datasets.load_dataset("starbucks_kfc_uk")
+    starbucks_locations = [(i[0], i[1]) for i in starbucks_kfc_locations if i[2] == 'starbucks'] 
+
     starbucks_layer = gmaps.symbol_layer(
         starbucks_locations, fill_color="green", stroke_color="green", scale=2
     )
```

Comment:
I think manipulating dataframes would fit in better with the rest of the documentation. The main users of `gmaps` seem to be data analysts or data scientists who are more comfortable with dataframes than list comprehensions. Something like (untested):

```python
df = gmaps.datasets.load_dataset_as_df("starbucks_kfc_uk")
starbucks_df = df[df["chain_name"] == "starbucks"]

starbucks_layer = gmaps.symbol_layer(
    starbucks_df[['latitude', 'longitude']], fill_color="green", stroke_color="green", scale=2
)
```